wiki/content/query-language/graphql-fundamentals.md

```diff
@@ -6,7 +6,7 @@ weight = 1
     parent = "query-language"
 +++
 
-Dgraph's DQL is based on Facebook's [GraphQL](https://facebook.github.io/graphql/).  GraphQL wasn't developed for Graph databases, but its graph-like query syntax, schema validation and subgraph shaped response make it a great language choice.  We've modified the language to better support graph operations, adding and removing features to get the best fit for graph databases.  We're calling this simplified, feature rich language, ''GraphQL+-''.
+Dgraph Query Language, _DQL_, ([previously named GraphQL+-](https://dgraph.io/blog/post/graphql+-to-dql/)) is based on Facebook's [GraphQL](https://facebook.github.io/graphql/).  GraphQL wasn't developed for Graph databases, but its graph-like query syntax, schema validation and subgraph shaped response make it a great language choice.  We've modified the language to better support graph operations, adding and removing features to get the best fit for graph databases.
```

Comment:
Let's remove the reference to Facebook - so it should say "is based on GraphQL" instead of "is based on Facebook's GraphQL"

Reply:
It now reads "...is based on GraphQL maintained by the GraphQL Foundation."

Reply:
@aaroncarey, is this issue resolved?

Reply:
Yep!